rules/windows/builtin/win_transferring_files_with_credential_data_via_network_shares.yml

```diff
@@ -0,0 +1,29 @@
+title: Transferring files with credential data via network shares
+description: Transferring files with well-known filenames (sensitive files with credential data) using network shares
+author: Teymur Kheirkhabarov, oscd.community
+date: 2019/10/22
+references:
+    - https://www.slideshare.net/heirhabarov/hunting-for-credentials-dumping-in-windows-environment
+tags:
+    - attack.credential_access
+    - attack.t1003
+logsource:
+    product: windows
+    service: security
+detection:
+    selection:
+        EventID: 5145
+        RelativeTargetName|contains:
+            - '\mimidrv'
+            - '\lsass'
+            - '\windows\minidump\'
+            - '\hiberfil'
+            - '\sqldmpr'
+            - '\sam'
+            - '\ntds.dit'
+            - '\security'
+    condition: selection
+falsepositives:
+    - Transferring sensitive files for legitimate administration work by legitimate administrator
+level: medium
+status: experimental
```

Comment:
don't know how to comment on the next rule which I've put to 'deprecated' — 
[win_susp_vssadmin_ntds_activity.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/deprecated/win_susp_vssadmin_ntds_activity.yml):

```
title: Activity Related to NTDS.dit Domain Hash Retrieval
status: experimental
description: Detects suspicious commands that could be related to activity that uses volume shadow copy to steal and retrieve hashes from the NTDS.dit file remotely
author: Florian Roth, Michael Haag
references:
    - https://www.swordshield.com/2015/07/getting-hashes-from-ntds-dit-file/
    - https://room362.com/post/2013/2013-06-10-volume-shadow-copy-ntdsdit-domain-hashes-remotely-part-1/
    - https://www.trustwave.com/Resources/SpiderLabs-Blog/Tutorial-for-NTDS-goodness-(VSSADMIN,-WMIS,-NTDS-dit,-SYSTEM)/
    - https://securingtomorrow.mcafee.com/mcafee-labs/new-teslacrypt-ransomware-arrives-via-spam/
    - https://dfironthemountain.wordpress.com/2018/12/06/locked-file-access-using-esentutl-exe/
tags:
    - attack.credential_access
    - attack.t1003
logsource:
    category: process_creation
    product: windows
detection:
    selection:
        CommandLine:
            - vssadmin.exe Delete Shadows
            - 'vssadmin create shadow /for=C:'
            - copy \\?\GLOBALROOT\Device\\*\windows\ntds\ntds.dit
            - copy \\?\GLOBALROOT\Device\\*\config\SAM
            - 'vssadmin delete shadows /for=C:'
            - 'reg SAVE HKLM\SYSTEM '
            - esentutl.exe /y /vss *\ntds.dit*
    condition: selection
fields:
    - CommandLine
    - ParentCommandLine
falsepositives:
    - Administrative activity
level: high
```

the reason for that is the creation of more specific, contextualised, detailed and optimised rules:

- [rules/windows/process_creation/process_creation_shadow_copies_creation.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/process_creation/process_creation_shadow_copies_creation.yml)
- [rules/windows/process_creation/process_creation_shadow_copies_deletion.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/process_creation/process_creation_shadow_copies_deletion.yml)
- [rules/windows/process_creation/process_creation_copying_sensitive_files_with_credential_data.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/process_creation/process_creation_copying_sensitive_files_with_credential_data.yml)
- [rules/windows/process_creation/process_creation_grabbing_sensitive_hives_via_reg.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/process_creation/process_creation_grabbing_sensitive_hives_via_reg.yml)